tests/phpunit/tests/test-widget-nav-menu.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+class Widget_Nav_Menu_Test extends PLL_UnitTestCase {
```

Comment:
Could you please clean up the global scope as done in other widget tests?
See https://github.com/polylang/polylang/blob/3.2.5/tests/phpunit/tests/test-widgets-filter.php#L34-L45